code/game/objects/structures/flora.dm

```diff
@@ -359,6 +362,27 @@
 		if(do_after(user,3 SECONDS,target=src))
 			to_chat(user,span_notice("You finish trimming [src]."))
 			change_visual()
+	// SKYRAT EDIT START - OOK CHEE
+	if(icon_state == "monkeyplant")
```

Comment:
i am aware why you have to do this but it's not any less disgusting

Reply:
yeah I hate it too. Might take it upstream but they don't even have any of the sprites and ehh. I'll come back to that one.